vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -239,12 +257,38 @@ class RoomDetailFragment :
         }
     }
 
+    private fun setupNotificationView() {
+        notificationAreaView.delegate = object : NotificationAreaView.Delegate {
+
+            override fun onTombstoneEventClicked(tombstoneEvent: Event) {
+                roomDetailViewModel.process(RoomDetailActions.HandleTombstoneEvent(tombstoneEvent))
+            }
+
+            override fun resendUnsentEvents() {
+                TODO("not implemented")
```

Comment:
will crash? maybe snackbar?

Reply:
Won't as it's not used, imported from Riot Legacy